WordPress/Classes/ViewRelated/Post/Search/PostSearchViewController.swift

```diff
@@ -18,6 +18,10 @@ final class PostSearchViewController: UIViewController, UITableViewDelegate, UIS
 
     private var cancellables: [AnyCancellable] = []
 
+    private var postDelegate: InteractivePostViewDelegate {
+        listViewController as! InteractivePostViewDelegate
```

Comment:
wdyt using as? instead?

Reply:
Great point. I simplified it [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/21917/commits/b43ab86f41e7fb10a70f0fe921646fffcd4b78cc) and removed the force unwrap. Looks like I also messed up the table delegate setup – fixed.